grakn/service/Session/TransactionService.py

```diff
@@ -152,6 +192,15 @@ def __next__(self):
     def __iter__(self):
         return self
 
+    def send_only(self, request):
+        if self._closed:
+            raise GraknError("This connection is closed")
+        try:
+            self._add_request(request)
+        except Exception as e:
+            self._closed = True
+            raise GraknError("Server/network error: {0}\n\n generated from request: {1}".format(e, request))
+
     def send(self, request):
```

Comment:
I think renaming to `send_receive`  is a good idea

Reply:
Agreed